pkg/rook/osd/osd.go

```diff
@@ -54,7 +54,7 @@ func SafeToDestroy(ctx context.Context, clientsets *k8sutil.Clientsets, operator
 	args := []string{"osd", "safe-to-destroy", osdID}
 	out, err := exec.RunCommandInOperatorPod(ctx, clientsets, "ceph", args, operatorNamespace, storageClusterNamespace, true)
 	if err != nil {
-		logging.Fatal(fmt.Errorf("failed to run ceph command with args %v. %v", args, err))
+		return false, errors.Wrapf(err, string(out))
```

Comment:
```suggestion
		return false, errors.Wrapf(err, string(out))
```
not sure if this is the right solution, as if previous job are running in the same cluster state maybe we need to filter the error message and continue removing.

Reply:
done